instrumentation/net/http/otelhttp/internal/semconv/v1.20.0.go

```diff
@@ -214,7 +213,7 @@ func (o OldHTTPClient) MetricAttributes(req *http.Request, statusCode int, addit
 
 	attributes := slices.Grow(additionalAttributes, n)
 	attributes = append(attributes,
-		standardizeHTTPMethodMetric(req.Method),
+		semconv.HTTPMethod(standardizeHTTPMethod(req.Method)),
```

Comment:
```suggestion
		standardizeHTTPMethod(req.Method),
```

Reply:
https://github.com/flc1125/opentelemetry-go-contrib/blob/fb25ee409b7c398366502223693f2a50909f7817/instrumentation/net/http/otelhttp/internal/semconv/util.go#L100-L108

Because the new and old logic are consistent. But the format of semconv is inconsistent. So I pulled it out.

Reply:
Yes, I'm not talking about the extraction of the method, but the use of `semconv.HTTPMethod`.
That's a nit though, so we can ship without this change.